### PR TITLE
Api usage

### PR DESCRIPTION
**Pull request by adjamenhmh**

### CODE REVIEW

**Comment by adjamenhmh** [comment]
@drochgenius Good catch, looking into it

File: package.json [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
     "devDependencies": {
         "@polymer/iron-demo-helpers": "^3.0.0-pre.19",
-        "@webcomponents/webcomponentsjs": "^2.0.0",
-        "growl": "^1.10.5",
-        "wct-browser-legacy": "^1.0.0"
+        "@webcomponents/webcomponentsjs": "^2.0.2",
+        "concurrently": "^3.6.0",
+        "typescript": "^2.9.2",
+        "wct-browser-legacy": "^1.0.0",
+        "node-sass": "^4.9.0"
```
Comment (drochgenius) [inline package.json]: missing colon

**Comment by drochgenius** [comment]
Actually @adjamenhmh , you are right, it works form `learning-spine-browser` and `spine-skill` elements.

THe issue is only for the `spine-node` element.
I always thought this `_render` function implementation is problematic.

I suggest the following: (1) let’s get your PR merged.
(2) Let’s see how you can refactor the `_render` function in a more standard and efficient and clean way.
What we observe point towards a bug in our code, rather than an issue with Polymer or Lit-Element itself.

**Comment by drochgenius** [comment]
Let's get'er merged!

**Pull request merged**